modules/ROOT/pages/provisioning-gcp.adoc

```diff
@@ -0,0 +1,120 @@
+= Provisioning Fedora bootc on GCP
+
+This guide shows how to provision new Fedora/CentOS bootc instances on the
+https://cloud.google.com/products/compute[Google Compute Engine] platform.
+
+== Prerequisites
+
+You'll need a Google Compute Engine account, and if following this full example, also https://opentofu.org/[OpenTofu].
+To be clear, the OpenTofu usage is just an example, you can provision instances
+however you like, including the https://cloud.google.com/sdk/gcloud[gcloud CLI]
+interactively in the console GUI, or in a Kubernetes environment
+using https://cluster-api.sigs.k8s.io/[Cluster API], etc.
```

Comment:
I think we should be consistent here.

Since we show the `aws` CLI being used to provision an AWS machine we should use the `gcloud` CLI for GCP.

We can certainly add examples with `TOFU` too since many people are familiar with it, but I'd advocate for consistency for our primary documentation for each cloud provider.

Reply:
I'd agree, I just happen to have written and tested the tofu one for gcp.  Not totally sure about having a "raw cli" and tofu variant for each one.  

It's complicated because ultimately of course there's at least 4-5 different primary ways one may want to provision - beyond those two there's also things like AWS CloudFormation, or on the other side k8s Cluster API etc...

Reply:
agree. in the FCOS docs we tried to just get by with the minimum needed to legitimately test deploying Fedora CoreOS on that cloud using the workflow documented by the cloud. we then plug this in to our [test day](https://github.com/coreos/fedora-coreos-tracker/issues/1685) events and have people "test the docs" with the latest images.

it would be a bit too much scope creep to try to doc all the different ways of deploying.